sklearn/metrics/_classification.py

```diff
@@ -1637,6 +1637,175 @@ def precision_recall_fscore_support(
+    likelihood ratios can be used to obtain post-test probabilities given a
+    pre-test probability.
+
+    ``LR+`` ranges from 1 to infinity. A ``LR+`` of 1 indicates that the
+    probability of predicting the positive class is the same for samples
+    belonging to either class; therefore, the test is useless. The greater
+    ``LR+`` is, the more a positive prediction is likely to be a true positive
+    when compared with the pre-test probability. A value of ``LR+`` lower than 1
+    is invalid as it would indicate that the odds of a sample being a true
+    positive decrease with respect to the pre-test odds.
+
+    ``LR-`` ranges from 0 to 1. The closer it is to 0, the lower the probability
+    of a given sample to be a false negative. A ``LR-`` of 1 means the test is
+    useless because the odds of having the condition did not change after the
+    test. A value of ``LR-`` greater than 1 invalidates the classifier as it
+    indicates an increase in the odds of a sample belonging to the positive
+    class after being classified as negative. This is the case when the
+    classifier systematically predicts the opposite of the true label.
+
+    A typical application in medicine is to identify the positive/negative class
+    to the presence/absence of a disease, respectively; the classifier being a
+    diagnostic test; the pre-test probability of an individual having the
+    disease can be the prevalence of such disease (proportion of a particular
+    population found to be affected by a medical condition); and the post-test
+    probabilities would be the probability that the condition is truly present
+    given a positive test result.
+
+    Read more in the :ref:`User Guide <class_likelihood_ratios>`.
+
+    Parameters
+    ----------
+    y_true : 1d array-like, or label indicator array / sparse matrix
+        Ground truth (correct) target values.
+
+    y_pred : 1d array-like, or label indicator array / sparse matrix
+        Estimated targets as returned by a classifier.
+
+    labels : array-like, default=None
+        List of labels to index the matrix. This may be used to select the
+        positive and negative classes with the ordering labels=[negative_class,
+        positive_class]. If None is given, those that appear at least once in
+        y_true or y_pred are used in sorted order.
+
+    sample_weight : array-like of shape (n_samples,), default=None
+        Sample weights.
+
+    raise_warning : bool, default=True
+        Whether or not a case-specific warning message is raised when there is a
+        zero division. Even if the error is not raised, the function will return
+        nan in such cases.
+
+    Returns
+    -------
+    (positive_likelihood_ratio, negative_likelihood_ratio) : tuple
+        A tuple of two float, the first containing the Positive likelihood ratio
+        and the second the Negative likelihood ratio.
+
+    Notes
```

Comment:
I am wondering if using a `Warns` section will be more appropriate as stated in the numpydoc.